docs/react-samples/package.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
The sample app is also a workspace. This will allow us to easily link updated widgets locally.

Reply:
All the files in this are demo, these will be updated when we add the actual implementation of these widgets. 
We can still run and see the basic providers and widgets working and sharing states.